Framework/Core/src/O2ControlHelpers.cxx

```diff
@@ -185,6 +185,43 @@ std::vector<RawChannel> extractRawChannels(const DeviceSpec& spec, const DeviceE
   return rawChannels;
 }
 
+bool isUniqueProxy(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::uniqueProxyLabel) != spec.labels.end();
+}
+
+bool shouldPreserveRawChannels(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::preserveRawChannelsLabel) != spec.labels.end();
+}
+
+bool isQcReconfigurable(const DeviceSpec& spec)
+{
+  return std::find(spec.labels.begin(), spec.labels.end(), ecs::qcReconfigurable) != spec.labels.end();
+}
+
+void dumpQcConfig(std::ostream& dumpOut, const DeviceExecution& execution, const DeviceSpec& spec, const std::string& indLevel)
```

Comment:
It could be renamed to `dumpProperties`, because if we are going to have to add another property, it will have to end up here.